rpcs3/Emu/System.cpp

```diff
@@ -1647,9 +1647,36 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 			{
 				if (!fs::is_file(g_cfg.vfs.get_dev_flash() + "sys/external/liblv2.sprx"))
 				{
-					if (!GetCallbacks().on_missing_fw())
+					const auto libs = g_cfg.core.libraries_control.get_set();
+
+					extern const std::map<std::string_view, int> g_prx_list;
+
+					// Check if there are any firmware SPRX which may be LLEd during emulation 
+					bool needs_fw = false;
+
+					for (const auto& lib : g_prx_list)
+					{
+						if (libs.count(std::string(lib.first) + ":lle") || (!lib.second && !libs.count(std::string(lib.first) + ":hle")))
```

Comment:
isn't the new c++20 thing to use .contains ?
Also, what is lib.second. there's no comment anywhere. not even where it's defined

Reply:
Not sure about compiler support. https://en.cppreference.com/w/cpp/compiler_support
Added comment.

Reply:
Idk if it matters, but in theory you could use a std::any_of if it supports maps.

Reply:
Done, it supports everything with iterator support.